src/index.html

```diff
@@ -4,13 +4,51 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="big-field"
+      action="google.com"
+      method="GET"
+      data-qa="input-big"
+     >
+
+      <input
+        type="search"
+        name="input-big"
+        placeholder="Try &quot;Los Angeles&quot;"
+        class="input-big"
+        data-qa="keypress"
+    >
+      <img src="images/Search.svg"
+           alt=""
+           class="search search-big"
+      >
```

Comment:
```suggestion
      <img
        src="images/Search.svg"
        alt=""
        class="search search-big"
      >
```
Fix formatting and don't leave `alt` attribute blank